index.bs

```diff
@@ -1693,6 +1737,15 @@ To <dfn>check if an [=attribution source=] can create [=aggregatable contributio
     |remainingAggregatableBudget|, return false.
 1. Return true.
 
+<h3 id="does-trigger-contain-aggregatable-data">Does trigger contain aggregatable data</h3>
```

Comment:
I'm not sure this heading is necessary: I don't think this minor algorithm needs to show up in the table of contents, and it already has an ID as part of the `<dfn>`.

Reply:
Yeah I thought about that as well, but this algorithm is used by two other algorithms, so I feel it's easier to organize this way. Wdyt?

Reply:
To me this algorithm just belongs in the general trigger section.

Reply:
Ok, moved the algorithm to the general trigger section, thanks.